docs/fe-style-guide.md

```diff
@@ -0,0 +1,76 @@
+# Bytebase Frontend Style Guide
+
+We are following [Bytebase's API Style Guide](https://github.com/bytebase/bytebase/blob/main/docs/api-style-guide.md) to ensure our style guide is **consistent**.
+
+# URL naming
+
+## Use slugs in URLs
+
+Bytebase uses [slugs](https://en.wikipedia.org/wiki/Clean_URL#Slug) in URLs for resource identification. We use `/issue/hello-world-101` instead of `/issue/101`
+
+_Rationale_: This makes URLs more clean and human-readable, especially when sharing via social media.
+
+## Use lower case, kebab-case for phrases
+
+Use `/anomaly-center` instead of `/anomalycenter` or `/anomalyCenter`
+
+_Rationale_: Using `/anomalycenter` makes it more difficult to read. `/anomalyCenter` makes it more difficult to memoize and type because of mixed cases. Using `/anomaly-center` is good for readability and improves consistency with slugs.
+
+## Use singular form even for collection resource
+
+Use `/issue` instead of `/issues` to display the list of issues.
+
+_Rationale_: Plural forms have several variations and it's hard for non-native English speakers to remember all the rules. And in practice, using singular form for collection resource won't cause confusion with the singular resource because they use different resource paths, e.g. `/issue` versus `/issue/:id`.
+
+# Naming
+
+## Variables and properties
+
+We are following the "Property Name Convention" guide in [Bytebase's API Style Guide](https://github.com/bytebase/bytebase/blob/main/docs/api-style-guide.md#property-name-convention).
+
+This allows us to follow vue recommended naming style such as [vue/attribute-hyphenation](https://eslint.vuejs.org/rules/attribute-hyphenation.html) and [vue/camelcase](https://eslint.vuejs.org/rules/camelcase.html). Making our code looks more likely to vue's "local accent".
+
+## Files and directories
```

Comment:
We do not mention the directories guide

Reply:
I've updated some rules and examples to directories. PTAL